### ISSUE TITLE
./bootstrap fails: libtoolize:   error: '/usr/share/aclocal/ltdl.m4' does not exist.

### ISSUE THREAD

**Issue by Qwertie-** [utsushi/utsushi#24]
Attempting to run ./bootstap on ubuntu 16.04 with all the recommended tools installed

```
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'upstream/aclocal'.
libtoolize: You should add the contents of 'upstream/aclocal/ltargz.m4' to 'aclocal.m4'.
libtoolize:   error: '/usr/share/aclocal/ltdl.m4' does not exist.
autoreconf: libtoolize failed with exit status: 1
configure.ac:236: error: possibly undefined macro: LT_CONFIG_LTDL_DIR
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
autoreconf: /usr/bin/autoconf failed with exit status: 1
./bootstrap: 282: ./bootstrap: cannot create upstream/ltdl/Makefile.am: Directory nonexistent
./bootstrap: 283: ./bootstrap: cannot create upstream/ltdl/Makefile.am: Directory nonexistent
automake: error: no Automake input file found for 'upstream/ltdl/Makefile'
automake: error: no input file found among supplied arguments
```

**Comment by paddy-hack** [comment]
I just ran `./install-deps` in a `ubuntu:16.04` Docker container (which is a _very_ minimal Ubuntu setup).  A `./bootstrap` after that worked fine but I noticed that the README doesn't mention the various _binary_ packages that distributions may be splitting _source_ packages into.

In your particular case, the `libtool` source package is mentioned in the README, but Ubuntu (and others) split that into a number of binary packages.  One of these is called `libtool` but you also need `libltdl-dev` for `./bootstrap` to succeed.

I guess I should put a note in the README as well as the `./bootstrap --help` output to clarify this.
